plugin-maven/src/main/java/com/diffplug/spotless/maven/json/ApplyJsonPatch.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.maven.json;
+
+import org.apache.maven.plugins.annotations.Parameter;
+
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.json.ApplyJsonPatchStep;
+import com.diffplug.spotless.maven.FormatterStepConfig;
+import com.diffplug.spotless.maven.FormatterStepFactory;
+
+/**
+ * A {@link FormatterStepFactory} implementation that corresponds to {@code <applyJsonPatch>...</applyJsonPatch>} configuration element.
+ */
+public class ApplyJsonPatch implements FormatterStepFactory {
+	private static final String DEFAULT_ZJSONPATCH_VERSION = "0.4.14";
+
+	@Parameter
+	String zjsonPatchVersion = DEFAULT_ZJSONPATCH_VERSION;
+
+	@Parameter
+	String patch;
```

Comment:
I wonder if we can do better here - initially I tried to use the same parameter type as the Gradle plugin, but unfortunately Maven can't deserialize a `List<Map<String, Object>>` as a `Parameter`, which is pretty much the closest representation of the JSON patch format we can get in Java types without hardcoding in a JSON AST.